src/NzbDrone.Core/Music/RefreshArtistService.cs

```diff
@@ -160,19 +160,20 @@ public void Execute(RefreshArtistCommand message)
             if (message.ArtistId.HasValue)
             {
                 var artist = _artistService.GetArtist(message.ArtistId.Value);
-                RefreshArtistInfo(artist);
+                RefreshArtistInfo(artist, false);
```

Comment:
Shouldn't this be true? Isn't this the call that is hit when you do a refresh on a single artist from the artist details page. Then below logic is when a complete library refresh is triggered.